sdk/eventhub/azure-eventhubs/azure/eventhub/eventprocessor/close_reason.py

```diff
@@ -0,0 +1,13 @@
+# --------------------------------------------------------------------------------------------
```

Comment:
We don't need one file per type.

Reply:
Moved this class to partition_processor.py